Lib/test/test_os.py

```diff
@@ -113,6 +113,12 @@ def test_getcwd(self):
         cwd = os.getcwd()
         self.assertIsInstance(cwd, str)
 
+    @unittest.skipUnless(hasattr(os, 'getdtablesize'), 'need os.getdtablesize()')
+    def test_getdtablesize(self):
+        size = os.getdtablesize()
+        self.assertIsInstance(size, int)
+        self.assertGreaterEqual(size, 0)
```

Comment:
It would be really silly if a process cannot open a single file. Sorry, I suggest to change the minimum to at least 1 file :-)

```suggestion
        self.assertGreaterEqual(size, 1)
```

Reply:
Hmm, you are right.